docs/source/dataset_summary.rst

```diff
@@ -42,6 +42,7 @@ Motor Imagery
         :class:`Weibo2014`,10,60,7,80,4s,200Hz,1,1,5600
         :class:`Zhou2016`,4,14,3,160,5s,250Hz,3,2,11496
         :class:`Stieger2021`,62,64,4,450,3s,1000Hz,7 or 11,1,250000
+        :class:`Liu2024`,50,29,2,40,4s,500Hz,1,1,160000
```

Comment:
Why 29 channels here and 30 in the PR description?

Reply:
Also, why 160000 trials in total? 50x40x2x1x1 = 4000

Reply:
At the begining of the PR I thought it was 30. It's fixed now. The trials are by now : 50*20*2 = 2000 if we don't include the break and instructions as classes in the description.